src/base/rss/rssdownloadrule.h

```diff
@@ -88,6 +88,9 @@ namespace Rss
         bool operator==(const DownloadRule &other) const;
 
     private:
+        bool matches(const QString &articleTitle, const QString &expression) const;
+
+    private:
```

Comment:
Did someone else told you do this? If not, then remove the 2nd `private:` label and just leave one empty line between the functions and variables.